packages/components/src/visually-hidden/test/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import TestRenderer from 'react-test-renderer';
+
+/**
+ * Internal dependencies
+ */
+import VisuallyHidden from '../';
+
+describe( 'VisuallyHidden', () => {
+	it( 'by default renders as a div with the visually hidden class name', () => {
```

Comment:
Two first tests duplicate unit tests generated from Storybook stories. See:

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/visually-hidden/stories/index.js

https://github.com/WordPress/gutenberg/blob/cf7f028c270ccabb19e3f2fca7e5098379b0dc6e/storybook/test/__snapshots__/index.js.snap#L8773-L8791

Do you think we should add more stories instead, or should those tests be more specif in how verification happens?

Reply:
Ah, I didn't actually know stories are snapshot tested! Makes absolute sense to use stories for these tests. I'll update the PR.